src/modules/mavlink/streams/HIL_ACTUATOR_CONTROLS.hpp

```diff
@@ -95,7 +95,7 @@ class MavlinkStreamHILActuatorControls : public MavlinkStream
 				    system_type == MAV_TYPE_OCTOROTOR ||
 				    system_type == MAV_TYPE_VTOL_DUOROTOR ||
 				    system_type == MAV_TYPE_VTOL_QUADROTOR ||
-				    system_type == MAV_TYPE_VTOL_RESERVED2) {
+				    system_type == MAV_TYPE_VTOL_FIXEDROTOR) {
```

Comment:
Where did these changes come in?

Reply:
It's cherry picked from here, to fix compile issues due to changes in Mavlink submodule!

https://github.com/PX4/PX4-Autopilot/pull/19375